{"name":"user1pass0","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [密码错误111] but found []","trace":"java.lang.AssertionError: expected [密码错误111] but found []\r\n\tat org.testng.Assert.fail(Assert.java:94)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:496)\r\n\tat org.testng.Assert.assertEquals(Assert.java:125)\r\n\tat org.testng.Assert.assertEquals(Assert.java:178)\r\n\tat org.testng.Assert.assertEquals(Assert.java:188)\r\n\tat com.ly.javaselenium.webdriver.testSuites.LoginTest.user1pass0(LoginTest.java:47)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:639)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:774)\r\n\tat org.testng.TestRunner.run(TestRunner.java:624)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:359)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:39)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)\r\n\tat org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)\r\n\tat java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\r\n\tat java.lang.Thread.run(Thread.java:748)\r\n"},"stage":"finished","description":"输入正确的用户名、错误的密码","steps":[{"name":"输入用户名、错误的密码","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1635129878723,"stop":1635129878724}],"attachments":[{"name":"失败截图","source":"8d5d07df-3b45-457e-aeb9-9991411a1801-attachment"}],"parameters":[],"start":1635129878718,"stop":1635129879880,"uuid":"3781d713-5a62-4395-a25d-0b8f3a98da08","historyId":"2ef2e0f154e3bf0f07fefee4e21f32bf","fullName":"com.ly.javaselenium.webdriver.testSuites.LoginTest.user1pass0","labels":[{"name":"package","value":"com.ly.javaselenium.webdriver.testSuites.LoginTest"},{"name":"testClass","value":"com.ly.javaselenium.webdriver.testSuites.LoginTest"},{"name":"testMethod","value":"user1pass0"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"D:/Java-work/JavaSelenium"},{"name":"subSuite","value":"com.ly.javaselenium.webdriver.testSuites.LoginTest"},{"name":"host","value":"liyanb-1"},{"name":"thread","value":"8912@liyanb-1.TestNG(13)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"输入操作"},{"name":"feature","value":"登录测试"}],"links":[]}